Issues when building

Running ``make build-devel`` always throws the following error and fails:
```
 ---> cc33ba558f1f
Step 26/46 : COPY --from=client/tools/qooxdoo-kit/builder:latest --chown=scu:scu   /project/build-output   /build/services/web/client

ERROR: for webserver  (<Service: webserver>, "invalid from flag value client/tools/qooxdoo-kit/builder:latest: pull access denied for client/tools/qooxdoo-kit/builder, repository does not exist or may require 'docker login': denied: requested access to the resource is denied")
ERROR: Service 'webserver' failed to build: invalid from flag value client/tools/qooxdoo-kit/builder:latest: pull access denied for client/tools/qooxdoo-kit/builder, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
Makefile:107: recipe for target 'build-devel' failed
make: *** [build-devel] Error 1
```
It would be cool and less time consuming to be able to run ``make build-devel`` without first having to ``make build``.

--- a/Makefile
+++ b/Makefile
@@ -105,7 +105,7 @@ build-nc: .env ## As build but w/o cache (alias: rebuild)
 .PHONY: build-devel
 build-devel: .env ## Builds development images and tags them as 'local/{service-name}:development'
 	# Compiling front-end
-	@$(MAKE) -C services/web/client compile-dev
+	@$(MAKE) -C services/web/client touch compile-dev
 	# Building services
 	@export BUILD_TARGET=development; \
 	$(DOCKER_COMPOSE) -f services/docker-compose.build.yml build --parallel

--- a/services/web/client/Makefile
+++ b/services/web/client/Makefile
@@ -28,10 +28,14 @@ compile-dev: qx_packages ## qx compiles host' 'source' -> host's 'source-output'
 			--set osparc.vcsStatusClient='"${VCS_STATUS_CLIENT}"' \
 			--set osparc.vcsOriginUrl='"${VCS_URL}"'
 
-.PHONY: compile
+.PHONY: compile touch
 compile: ## qx compiles host' 'source' -> image's 'build-output'
 	# qx compile 'source' within $(docker_image) image  [itisfoundation/qooxdoo-kit:${QOOXDOO_KIT_TAG}]
-	@docker build -f $(docker_file) -t $(docker_image) --build-arg tag=${QOOXDOO_KIT_TAG} .
+	@docker build -f $(docker_file) -t $(docker_image) --build-arg tag=${QOOXDOO_KIT_TAG} --target=build-client .
+
+touch: ## minimal image build with /project/output-build inside
+	# touch /project/output-build such that multi-stage 'services/web/Dockerfile' can build development target (fixes #1097)
+	@docker build -f $(docker_file) -t $(docker_image) --build-arg tag=${QOOXDOO_KIT_TAG} --target=touch .
 
 
 # qx serve --------------------------

--- a/services/web/client/tools/qooxdoo-kit/builder/Dockerfile
+++ b/services/web/client/tools/qooxdoo-kit/builder/Dockerfile
@@ -4,10 +4,14 @@
 # Note: context at osparc-simcore/services/web/client expected
 #
 ARG tag
-FROM itisfoundation/qooxdoo-kit:${tag} as build-client
+FROM itisfoundation/qooxdoo-kit:${tag} as touch
 
 WORKDIR /project
 ENV PATH=/home/node/node_modules/.bin:${PATH}
+
+RUN mkdir /project/build-output
+
+FROM touch as build-client
 
 # Installs contributions
 COPY --chown=node:node compile.json compile.json

